index.html

```diff
@@ -13,13 +13,13 @@
 </head>
 
 <body>
-  
+
   <header>
     <h1>Bills Sonoma</h1>
 	<div onclick="" class="help-tip">
 	  <p>
-		Bills Sonoma is a band :~) <br>
```

Comment:
why did we remove this?

Reply:
lol i removed the smiley it was bothering me idk why